supportfiles/.condarc

```diff
@@ -0,0 +1,3 @@
+channels:
+  - conda-forge
+  - defaults
```

Comment:
I think we want to remove `defaults` since it usually causes problems with the solver and other things (sorry I don't have specifics).  Can you remove and re-test?